qfepweb/settings.py

```diff
@@ -135,7 +135,8 @@ def void():
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = BASE_DIR / "static"
```

Comment:
This was my mistake: uploaded user files goes to MEDIA, not STATIC. Static is reserved to webpage assets (such as images, css, javascript and so on).